integration-libs/cdc/root/service/cdc-js.service.spec.ts

```diff
@@ -620,6 +620,34 @@ describe('CdcJsService', () => {
     });
   });
 
+  describe('handleProfileUpdateTostMessage', () => {
```

Comment:
Correct the method name as mentioned below

Reply:
Done